pkg/pillar/cmd/volumemgr/blob.go

```diff
@@ -104,6 +116,195 @@ func downloadBlob(ctx *volumemgrContext, objType string, sv SignatureVerifier, b
 	return changed
 }
 
+//loadBlobs loads the list of blob into CAS and removes verifyImage reference after loading the blob.
+func loadBlobs(ctx *volumemgrContext, blobs []*types.BlobStatus) error {
```

Comment:
I don't think this should go here. Loading a bunch of blobs into a CAS is a CAS thing, and should be exposed by an API, but the implementation is specific to the individual CAS. This should be wrapped in the `cas` package, and implemented in the `containerdCAS` instance.

It also makes all of the labels and such inside volumemgr, which is not the right place for it.

Reply:
Also, how do you load the image? You need that to keep the blobs from being garbage collected. The image points to the root blob, which points to the others.

Reply:
Modified CAS `IngestBlob` API and moved the steps handled in `loadBlobs` to `cas/containerd.go:IngestBlob()`